packages/kbn-config-schema/src/types/type.ts

```diff
@@ -119,7 +137,8 @@ export abstract class Type<V> {
   }
 
   /**
-   * @internal
+   * @note intended for internal use, if you need to use this please contact
+   *       the core team to discuss your use case.
```

Comment:
This was causing type check issues because the built types did not include `getSchema`. Surprising that this was not leading to an issue before now.